eslint.config.mjs

```diff
@@ -24,6 +24,10 @@ const config = createConfig([
       // e.g. `txreceipt_status`, `signTypedData_v4`, `token_id`
       camelcase: 'off',
       'id-length': 'off',
+      'n/no-unsupported-features/node-builtins': 'off',
```

Comment:
Can we move these changes to the `packages/foundryup` block you've added? I can work to consolidate these later, just didn't want to add global overrides if we didn't have to.